### PR TITLE
Feature fileupload

### PR DESCRIPTION
**Pull request by Tugi-Bugi**

### CODE REVIEW

File: src/main/java/models/manager/FileManager.java [hunk 1]
```diff
@@ -1,5 +1,270 @@
...
+    @Transactional
+    private String createPictureUser(String email, String fileName) {
+        log.log(Level.INFO, "Start add ProfilePicture to User in Database");
+
+        EntityManager entityManager = entitiyManagerProvider.get();
+        PeasyUser peasyUser = entityManager.find(PeasyUser.class, email);
+
+        ProfilePicture profilePicture = new ProfilePicture(fileName);
+        profilePicture.setPeasyUser(peasyUser);
+        peasyUser.setProfilePicture(profilePicture);
```
Comment (Nijin22) [inline src/main/java/models/manager/FileManager.java]: Please add error handling when `email` does not identify a user. (This happens when peasyUser == null)

File: src/main/java/models/manager/FileManager.java [hunk 2]
```diff
@@ -1,5 +1,270 @@
...
+            }
+            if (os != null) {
+                os.flush();
+                os.close();
+            }
+        }
+        return saveFile.getCanonicalPath();
+
+    }
+
```
Comment (Nijin22) [inline src/main/java/models/manager/FileManager.java]: Where is the corresponding database entry (`ProfilePicture`, `ProjectFile`, ...) deleted from the database? Shouldn't this happen here?

File: src/main/java/models/manager/ProjectManager.java [hunk 3]
```diff
@@ -62,7 +62,7 @@ public Project createProject(PeasyUser projectManager, String name) throws Illeg
      * @return Project
      * @throws NoSuchElementException
      */
-    @UnitOfWork
+    @Transactional
```
Comment (Tugi-Bugi) [inline src/main/java/models/manager/ProjectManager.java]: nein, das war unit of work aber da gab es einenProblem, es konnte die Daten nicht lesen
in google habe ich mal geschaut und es haben sehr viele dieses Problem und die Lösung ist dann die Methode mit transactional zu annotieren

File: src/main/java/models/manager/FileManager.java (hunk 2 above)
Comment (Tugi-Bugi) [inline src/main/java/models/manager/FileManager.java]: no, this is happening in mehtods createPictureUser and deletePictureUser

Comment (Nijin22) [inline src/main/java/models/manager/FileManager.java]: In that case, I suggest either making this a private method or documenting very clearly what methods need to be called to remove a profile picture

Comment (Tugi-Bugi) [inline src/main/java/models/manager/FileManager.java]: I am going to do this now, but I wanted to wait for your feedback before createing the documetation for the methods

**Pull request merged**